### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreyc0d3r**
This is a very nasty gotcha that can suck out hours from your day.

## Checklist:
- [ ] My code is tested.
- [ ] My code follows the WordPress code style. 
- [ ] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation. 
- [ ] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/block-editor/src/components/inner-blocks/README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 
 - `all` — prevents all operations. It is not possible to insert new blocks. Move existing blocks or delete them.
```
Comment (talldan) [inline packages/block-editor/src/components/inner-blocks/README.md]: Could also add quotation marks around the text to make it clearer that the first two options are strings.

```suggestion
- `'all'` — prevents all operations. It is not possible to insert new blocks. Move existing blocks or delete them.
```

File: packages/block-editor/src/components/inner-blocks/README.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ Template locking allows locking the `InnerBlocks` area for the current template.
...
 - `insert` — prevents inserting or removing blocks, but allows moving existing ones.
```
Comment (talldan) [inline packages/block-editor/src/components/inner-blocks/README.md]:
```suggestion
- `'insert'` — prevents inserting or removing blocks, but allows moving existing ones.
```

File: packages/block-editor/src/components/inner-blocks/README.md [hunk 3]
```diff
@@ -107,7 +107,7 @@ Template locking allows locking the `InnerBlocks` area for the current template.
...
-- `false` — prevents locking from being applied to an `InnerBlocks` area even if a parent block contains locking.
+- `false` — prevents locking from being applied to an `InnerBlocks` area even if a parent block contains locking. ( Boolean )
```
Comment (talldan) [inline packages/block-editor/src/components/inner-blocks/README.md]: How about making it more prominent?

```suggestion
- `false` — (Boolean) prevents locking from being applied to an `InnerBlocks` area even if a parent block contains locking.
```

**Review by talldan — APPROVED** [review]
Thanks for the PR @andreyc0d3r! I'm pretty happy with the change, though I left a few suggestions to make it even more noticable.

File: packages/block-editor/src/components/inner-blocks/README.md (hunk 3 above)
Comment (andreyc0d3r) [reply packages/block-editor/src/components/inner-blocks/README.md]: Looks great!

**Pull request merged**